db.go

```diff
@@ -877,7 +877,7 @@ func (db *DB) openHeadBlock(dir string) (*HeadBlock, error) {
 	)
 	wal, err := OpenSegmentWAL(wdir, l, 5*time.Second)
 	if err != nil {
-		return nil, errors.Wrap(err, "open WAL %s")
+		return nil, errors.Wrap(err, "open WAL %s", dir)
```

Comment:
Oops, sorry! That happens when a fix looks too easy. Force-pushed an update now, and ran this through essentially the .travis.yml commands locally.